### PR TITLE
fixed a small issue in Cursor

### PR DESCRIPTION
**Pull request by velikaww**
if cursor got disabled and enabled later at another surface, state may not appear correctly as the common sense. this is because target get set to null when disabled but not recover/update at enable.

### CODE REVIEW

File: Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs [hunk 1]
```diff
@@ -190,6 +197,8 @@ protected virtual void OnDisable()
             TargetedObject = null;
             TargetedCursorModifier = null;
             visibleHandsCount = 0;
+            IsHandVisible = false;
+            OnCursorStateChange(CursorStateEnum.Contextual);
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs]: Would we just want to change to context to None here as well?
What purpose does contextual have?

Comment (velikaww) [inline Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs]: I just keep consist with what OnInputDisabled does

Comment (StephenHodgson) [inline Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs]: I was just curious is all, thanks.

Comment (velikaww) [inline Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs]: fine:)

**Comment by velikaww** [comment]
@StephenHodgson The PR will be merges automatically or requires me some actions? or someone else (e.g. you) manually do it?

**Comment by StephenHodgson** [comment]
Pull requests need a minimum of two reviews.

**Pull request merged**